src/common/search/FilterPlays.jsx

```diff
@@ -186,18 +194,17 @@ const FilterPlays = () => {
 
       <button
         onClick={() => setShowModal(true)}
-        className="btn-filter"
-        title="Filter Plays"
-      >{
-          noOfAppliedFilter === 0 ?
-            null
-            : <div className="badge">{noOfAppliedFilter}</div>
-        }
+        className='btn-filter'
+        title='Filter Plays'
+      >
+        {noOfAppliedFilter === 0 ? null : (
+          <div className='badge'>{noOfAppliedFilter}</div>
+        )}
 
         <RiFilterFill
-          className="icon"
-          size="28px"
-          color="var(--color-neutral-30"
+          className='icon'
+          size='28px'
+          color='var(--color-neutral-30'
```

Comment:
Here is a typo and incomplete expression.

Reply:
@Angryman18  pls take care

Reply:
fixed